scripts/background.js

```diff
@@ -1,10 +1,16 @@
+/** Persistent background script.
+ * MAYBE: Migrate to non-persistent, event-based background script.
+ * info: https://developer.chrome.com/extensions/background_migration
+ */
+
 // Update check
 chrome.runtime.onUpdateAvailable.addListener(details => {
-    console.log("MHHH: updating to version " + details.version);
+    console.log(`MHHH: updating to version ${details.version}`);
     chrome.runtime.reload();
 });
 
-var time_interval = 7200 * 1000; // seconds * 1000
+// TODO: Do we need to request the update at all? Chrome auto-checks automatically.
```

Comment:
we do not need to request update at all, but i'm not sure how fast Opera works. I did this originally to make sure the addon updates asap. We can take this code out. good catch.